SUPPORT.md

```diff
@@ -0,0 +1,22 @@
+# Support
+
+The issue tracker is reserved for reporting bugs and we don't provide support through it. Support tickets opened via the issue tracker will be closed.
+
+If you require support there are two types of support available, community support and commercial support.
+
+## Community support
+
+Community support is provided by members of our open source community and is provided out of their good will. Please be kind, polite and corteous to those that try to help you.
+
+You can find support via:
+
+- [Stack Overflow (remember to tag as silverstripe)](http://stackoverflow.com/questions/tagged/silverstripe)
+- [Slack](http://silverstripe.org/slack)
+
+For more information please see our [community page](http://www.silverstripe.org/community/#introduction).
+
+## Commercial support
+
+If you require a higher level of support with an SLA, you may want to consider commercial support provided by SilverStripe Ltd.
```

Comment:
I think that's too one-sided - it should point to https://www.silverstripe.org/community/developer-and-partner-directory/.